manifests/klusterlet/management/klusterlet-registration-clusterrolebinding-addon-management.yaml

```diff
@@ -0,0 +1,13 @@
+# Permissions for registration agent to manage Hosted mode addons
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: open-cluster-management:management:{{ .KlusterletName }}-registration:addon-management
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: open-cluster-management:management:{{ .KlusterletName }}-registration::addon-management
+subjects:
```

Comment:
same comments

Reply:
but the namespaces of the subjects service account are different, one is `{{ .AgentNamespace }}`(klusterlet name in Hosted mode), the other is `{{ .KlusterletNamespace }}`(klusterlet.spec.namespace)